source/renderer/app/components/wallet/receive/WalletReceiveDialog.js

```diff
@@ -383,9 +383,11 @@ export default class WalletReceiveDialog extends Component<Props, State> {
     return (
       <Dialog
         title={intl.formatMessage(messages.dialogTitle)}
+        walletName={walletName}
```

Comment:
@aleksandardjordjeviciohk please try to re-use the `subtitle` property instead of adding a new one `walletName` here 🙏